src/orb.yml

```diff
@@ -215,7 +215,7 @@ executors:
         default: '27'
     environment:
       FL_OUTPUT_DIR: output
-      _JAVA_OPTIONS: '-Xmx1500m'
+      _JAVA_OPTIONS: "-XX:MaxRAM=2048m -Xmx1024m"
```

Comment:
mind adding a small comment saying exactly what this does?

Reply:
Circle CI is unable to detect the amount of memory within a *container*. Generally you could use an option like `UseCGroupMemoryLimitForHeap` which would help regulate the heap size to not overuse allocated memory.

In this case our container has been allowing the heap to exceed the allocated memory (4g) due to the container not knowing the actual allocation amount.

The `XX:MaxRAM` arg allows us to tell the container a max memory size it can use, which will allow the heap will to stay within bounds and not "kill" itself by overusing allocated memory.

Here is the result of progresively decreasing the MAX Ram and Heap Size on CI:

| MAX RAM (MB)  | HEAP (MB) | TIME |
| ------------- | ------------- | ------ |
|2048 | 1024 | 2:56 |
|1024 |  512 | 3:11 |
| 512 | 256 | OOM - FAILED  |